network/src/network.rs

```diff
@@ -1275,6 +1288,11 @@ impl NetworkController {
     pub fn protocols(&self) -> Vec<(ProtocolId, String, Vec<String>)> {
         self.network_state.protocols.read().clone()
     }
+
+    pub fn ping_peers(&self) {
```

Comment:
why not mut here？

Reply:
need to change the filed of `NetworkController` to `Arc` if we using mut here:
pub struct NetworkController {
    ping_controller: Arc<Sender<()>>,
}

Reply:
it's limited by rpc macro:
https://github.com/nervosnetwork/ckb/blob/670bdddbf343b639a22fe2c06a545fd45b522c6b/rpc/src/module/net.rs#L58

Reply:
I probably know the reason its trait requires send + sync